.github/workflows/image.yml

```diff
@@ -0,0 +1,84 @@
+name: Release Images
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: 'Infinispan Server version'
+        required: true
+        type: string
+      branch:
+        description: 'The repository branch to checkout'
+        required: true
+        type: string
+      type:
+        description: 'The type of release (default, latest, prerelease)'
+        required: true
+        type: string
+      push:
+        description: 'Push to remote repositories'
+        default: true
+        type: boolean
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Infinispan Server version'
+        required: true
+        type: string
+      branch:
+        description: 'The repository branch to checkout'
+        default: main
+        type: string
+      type:
+        description: The type of release
+        required: true
+        default: default
+        type: choice
+        options:
+          - default
```

Comment:
What's the significance of the `default` type, shouldn't this just be `latest`?

Reply:
It comes from the main release workflow, where we mix old and new releases, and latest only applies to the release from the current stable branch. I should normalize this to `latest` for the images